### PR TITLE
[O11y][HAProxy] Update misleading visualizations on haproxy.stat.connection.total field and add new visualizations.

### PR DESCRIPTION
**Pull request by harnish-elastic**
- Bugfix

## Proposed commit message

- Update the description of the field haproxy.stat.connection.total that providing number of connections for frontend
- There are OOTB dashboards visualizations for backend and server connections need to be removed. Updated all the dashboards with the proper filter and removed not relevant visualizations! Please refer [img](https://github.com/elastic/integrations/pull/11401/files#diff-d4a1cab2fc83d93022f5f1fd5e8ec5caca533575fe8f662f7b218dba2a99ac35) folder to compare the dashboards.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).
- [x] I have verified that any added dashboard complies with Kibana's [Dashboard good practices](https://docs.elastic.dev/ux-guidelines/data-viz/dashboard-good-practices) 

## Related issues

- Relates #11330

### CODE REVIEW

File: packages/haproxy/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.14.1"
+  changes:
+    - description: Update misleading visualizations on haproxy.stat.connection.total field and add new visualizations.
```
Comment (muthu-mps) [inline packages/haproxy/changelog.yml:4]:
```suggestion
    - description: Fix haproxy.stat.connection.total field and update visualizations.
```

File: packages/haproxy/data_stream/stat/fields/fields.yml [hunk 2]
```diff
@@ -72,7 +72,7 @@
           type: long
           metric_type: counter
           description: |
-            Cumulative number of connections.
+            Cumulative number of frontend connections.
```
Comment (muthu-mps) [inline packages/haproxy/data_stream/stat/fields/fields.yml:75]: Have you gone through the documentation to confirm that this metric is for the frontend connections?

**Comment by muthu-mps** [comment]
## Dashboard comments
+ Why do we have `haproxy.stat.component_type : 1` the query filter? What is the differentiation factor? Are we capturing those in the document to make sure that the connection type value differentiate the frontend, backend etc...
+ Backend dashboard - Change the title `Downtime seconds` to Downtime (in seconds) or Downtime. Same applies for HTTP backend dashboard as well.
+ Frontend dashboard - `Trafic volume overtime` to Network trafic overtime.
+ HTTP Servers - Active Servers has multiple mentions. Hide the title if possible.

File: packages/haproxy/data_stream/stat/fields/fields.yml (hunk 2 above)
Comment (harnish-elastic) [reply packages/haproxy/data_stream/stat/fields/fields.yml:75]: Yes please refer to this [link](https://www.haproxy.org/download/2.0/doc/management.txt#:~:text=known%20conn_rate%0A%2079%3A-,conn_tot,-%5B.F..%5D%3A%20cumulative%20number). 
```
79: conn_tot [.F..]: cumulative number of connections

L (Listeners), F (Frontends), B (Backends), and S (Servers).
```

**Comment by harnish-elastic** [comment]
> ## Dashboard comments
> * Why do we have `haproxy.stat.component_type : 1` the query filter? What is the differentiation factor? Are we capturing those in the document to make sure that the connection type value differentiate the frontend, backend etc...

Please checkout [this](https://github.com/elastic/integrations/blob/main/packages/haproxy/data_stream/stat/fields/fields.yml#L22) field description. It says that `Component type (0=frontend, 1=backend, 2=server, or 3=socket/listener)`. In some of the panel, the filter was already there. So we have moved it to the dashboard level to follow best practice.

> * Backend dashboard - Change the title `Downtime seconds` to Downtime (in seconds) or Downtime. Same applies for HTTP backend dashboard as well.

Sure I will make this change.

> * Frontend dashboard - `Trafic volume overtime` to Network trafic overtime.

Sure I will make this change.

> * HTTP Servers - Active Servers has multiple mentions. Hide the title if possible.

Sure, let me check if it looks good.

**Comment by harnish-elastic** [comment]
@muthu-mps Addressed review comments. Thank you!

**Review by muthu-mps — APPROVED** [review]
LGTM!

**Pull request merged**